test/gaudi_opts/geoTest_cld.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2019-2023 Key4hep-Project.
+#
+# This file is part of Key4hep.
+# See https://key4hep.github.io/key4hep-doc/ for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from Gaudi.Configuration import *
+
+from Configurables import TrackingCellIDEncodingSvc, GeoSvc
+algList = []
+svcList = []
+
+geoservice = GeoSvc("GeoSvc")
```

Comment:
side note: we have to be very careful here with the type `GeoSvc`, which comes from k4FWCore here. k4ActsTracking defines the same name again. I guess we need to make an eventual cleanup.

Reply:
Acts GeoSvc renaming is already in the works: key4hep/k4ActsTracking#12